--- SDL2-2.0.12/build/org.sthaid.reversi/app/jni/src/Android.mk	2020-09-14 19:51:27.650129668 -0400
+++ new/build/org.sthaid.reversi/app/jni/src/Android.mk	2020-09-13 15:17:46.776566156 -0400
@@ -5,13 +5,17 @@
 LOCAL_MODULE := main
 
 SDL_PATH := ../SDL
+SDL_TTF_PATH := ../SDL2_ttf-2.0.15
+
 
-LOCAL_C_INCLUDES := $(LOCAL_PATH)/$(SDL_PATH)/include
+LOCAL_C_INCLUDES := $(LOCAL_PATH)/$(SDL_PATH)/include \
+                    $(LOCAL_PATH)/$(SDL_TTF_PATH) 
+
 
 # Add your application source files here...
-LOCAL_SRC_FILES :=  stub.c
+LOCAL_SRC_FILES :=  main.c human.c cpu.c old.c game_utils.c util_sdl.c util_misc.c version.c
 
-LOCAL_SHARED_LIBRARIES := SDL2
+LOCAL_SHARED_LIBRARIES := SDL2 SDL2_ttf
 
 LOCAL_LDLIBS := -lGLESv1_CM -lGLESv2 -llog
 
--- SDL2-2.0.12/build/org.sthaid.reversi/app/src/main/res/values/strings.xml	2020-09-14 19:51:27.644129724 -0400
+++ new/build/org.sthaid.reversi/app/src/main/res/values/strings.xml	2020-09-13 19:38:52.284118088 -0400
@@ -1,3 +1,3 @@
 <resources>
-    <string name="app_name">Game</string>
+    <string name="app_name">Reversi</string>
 </resources>
